tests/test_flow.py

```diff
@@ -2556,6 +2556,21 @@ def test_custom_string_time_input():
     assert result.time == datetime(2021, 5, 9, 14, 5, 27, tzinfo=pytz.utc)
 
 
+def test_none_key_is_not_written():
+    data = pd.DataFrame({'first_name': ['moshe', None, 'katya'], 'some_data': [1, 2, 3]})
+    data.set_index(keys=['first_name'], inplace=True)
+
+    controller = build_flow([
+        DataframeSource(data, key_field=['first_name']),
+        Reduce([], append_and_return),
+    ]).run()
+    result = controller.await_termination()
+    expected = [{'first_name': 'moshe', 'some_data': 1}, {'first_name': 'katya', 'some_data': 3}]
+
+    assert result == expected, \
+        f'actual did not match expected. \n actual: {result} \n expected: {expected}'
```

Comment:
No need for this - pytest already prints this stuff. I know some tests (aggr...) do this, but none of the tests in this file do.